### DISCUSSION TITLE
可否加入按下alt再做手势可以临时绕过排除限制?

### DISCUSSION THREAD

**Discussion by rsyqvthv** [shuax/MouseInc.Settings#160] [Ideas]
我很喜欢mouseinc的手势贴图, 我不需要很重的功能, 有了它就立刻替换了snipaste. 
但这让我在那些放在排除列表中的程序无法使用mouseinc中的手势贴图. 
而这些软件也确实难以从排除列表中移除, 因为他们的右键操作比较多.

所以我想能否在"排除列表"中加一个临时绕过的快捷键, 比如alt, 按下它再做手势, 就可以临时性的绕过排除限制. 
这样我就可以在这些被排除的程序中使用"手势贴图"功能了, 甚至一些临时的手势操作. 

但我想这也许有点儿难度, 因为按住alt绕过排除后, 再做手势输出快捷键, 会不会让输出的快捷键多加了一个alt的按键?
我对此了解的不多, 也许有其他的按键可以作为绕过键, 这样也是可以的.

另外, 我也看到有其他朋友有关于排除列表的想法:

> 希望增加按住辅助键如Alt时临时禁用右键手势 https://github.com/shuax/MouseInc.Settings/discussions/99
> 可以设置指定软件使用，其他软件不使用手势吗 https://github.com/shuax/MouseInc.Settings/discussions/145

我想其实综合的看, 就是一个: 黑白名单的设置切换(要么黑名单, 要么白名单) 和 一个临时绕过名单的快捷键. 

我想, 无论黑名单还是白名单, "临时绕过的快捷键"似乎都是一个挺必要的补充. 
真心希望作者能够加上这个功能. 十分感谢.

**Comment by shuax** [comment]
大概可以用快捷键贴图

**Reply by rsyqvthv** [reply]
感谢你提供给我的方法. 

但仍然希望作者能考虑加入"迂回键"
除了截图外, 如果能在那些平时限制的软件中临时的解除限制, 进行手势操作, 这样会尽可能的萃取mouseinc的效率能量.
我想这是其他手势工具所不能及的.

当然, 
非常感谢作者能提供这么优秀的手势工具, 
我提交的需求并无强迫之意, 只是尽可能说明需求的利弊, 
最终是否加入, 当然作者决定, 悉听尊便. 

:)
